app.js

```diff
@@ -37,6 +40,14 @@ app.use((err, req, res, next) => {
   }
 });
 
+app.use((err, req, res, next) => {
+  if (err.code === '23503') {
+    res.status(404).send({ msg: 'Article not found' });
+  } else {
+    next(err);
+  }
+});
+
```

Comment:
I think you should start to consider moving these into a separate file now - they're becoming a big part of the app.js and they don't need to be :)